chainer_chemistry/dataset/parsers/csv_file_parser.py

```diff
@@ -13,6 +13,18 @@
 import traceback
 
 
+def _read_csv(filepath):
+    try:
+        # It is recommended to use `read_csv` method in pandas version
+        # after 0.18.x
+        df = pandas.read_csv(filepath)
```

Comment:
It seems Pandas v0.15.0 already has `read_csv` (See [release note](https://github.com/pandas-dev/pandas/releases/tag/v0.15.0)). I thought it is safe to assume Pandas in most environment has this method, as v0.15.2. But let me know if you know some common environment in which this assumption does not hold.
 

```
In [1]: import pandas as pd
In [2]: pd.read_csv
Out[2]: <function pandas.io.parsers.read_csv>
In [3]: pd.__version__
Out[3]: '0.15.2'
```